opencti-platform/opencti-graphql/src/modules/indicator/indicator-domain.ts

```diff
@@ -231,6 +233,10 @@ export const addIndicator = async (context: AuthContext, user: AuthUser, indicat
   // find default decay rule (even if decay is not activated, it is used to compute default validFrom and validUntil)
   const decayRule = await findDecayRuleForIndicator(context, observableType);
   const { validFrom, validUntil, revoked, validPeriod } = await computeValidPeriod(indicator, decayRule.decay_lifetime);
+  const extractedObservableValues = getObservableValuesFromPattern(formattedPattern);
+  if (!extractedObservableValues.length) {
+    throw FunctionalError(`Indicator of type ${indicator.pattern_type} is not correctly formatted.`);
```

Comment:
really ? do we always extract observable values from indicator pattern ? if we don't find observable values, we don't create the indicator ? extraction works only with stix pattern, what about other pattern types ?

Reply:
also I think it would be best to make sure we don't extract multiple times these observable values, like what we do in createObservablesFromIndicator.